tools/Raven.Debug/LogTrafficWatch/TrafficWatchReplay.cs

```diff
@@ -278,6 +285,29 @@ private static string ConvertJsonElement(JsonElement e)
                     _ => $"\"{e}\""
                 };
             }
+
+            private static void OutputLogToConsole(HttpResponseMessage response, byte[] responseBuffer, TrafficWatchHttpChange item)
+            {
+                Console.WriteLine("Failed to execute request");
+                
+                Console.WriteLine($"Http method: {item.HttpMethod}");
+                
+                Console.WriteLine($"Request Uri: {item.RequestUri}");
+                
+                Console.WriteLine($"Absolute Uri: {item.AbsoluteUri}");
+                        
+                Console.WriteLine($"Custom info: {item.CustomInfo}");
+                        
+                Console.WriteLine($"Status code: {(int)response.StatusCode}");
+                
+                Console.WriteLine($"Request size in bytes: {item.RequestSizeInBytes}");
+                
+                Console.WriteLine($"Response size in bytes: {item.ResponseSizeInBytes}");
+                
+                Console.WriteLine($"Request Id: {item.RequestId}");
+                    
+                Console.WriteLine(System.Text.Encoding.Default.GetString(responseBuffer));
```

Comment:
Let's use `StringBuilder` here and do single `Console.WriteLine()`

Reply:
Changed